feder/monitorings/migrations/0023_monitoring_hide_new_cases.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 2.2.27 on 2022-07-30 17:17
```

Comment:
Tu powinna być tylko jedna migracja. Zamiast tego masz migracje, gdzie to samo pole dodajesz, usuwasz, dodajesz.